Vagrantfile

```diff
@@ -0,0 +1,17 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+Vagrant.configure('2') do |config|
+  config.vm.box = 'ubuntu/trusty64'
```

Comment:
Currently edx still runs on precise so we should do the development on precise for now so we don't have compatibility issues.